packages/utils/adapters/src/adapters.ts

```diff
@@ -1,10 +1,11 @@
 import { TextProps, ViewProps, ImageProps } from 'react-native';
 import { IFilterMask } from './filter.types';
-import type { ITextWin32Props } from '@office-iss/react-native-win32';
+import type { ITextWin32Props, IViewWin32Props } from '@office-iss/react-native-win32';
+import { IViewWindowsProps } from 'react-native-windows';
 
 // export core interface types
 export type ITextProps = TextProps & Partial<ITextWin32Props>;
-export type IViewProps = ViewProps;
+export type IViewProps = ViewProps & Partial<IViewWin32Props> & Partial<IViewWindowsProps>;
```

Comment:
There are some macOS specific props that are not included in here as well, like `nextKeyView`

Reply:
I don't see a macOS adapter file, so that might be why. Do we need to create one?

Reply:
Oh yeah, there isn't one. Probably, since this package seems pretty important and we're starting to use a lot of controls on macOS